modules/bitgo/test/v2/unit/coins/utxo/address.ts

```diff
@@ -10,6 +10,7 @@ import { AbstractUtxoCoin, GenerateAddressOptions } from '@bitgo/abstract-utxo';
 
 import { utxoCoins, keychains as keychainsBip32, getFixture, shouldEqualJSON } from './util';
 
+// TODO (@rushilbg): Delete these tests because they are redundant (similar tests are in utxo-lib)
```

Comment:
Should we make a ticket to track this TODO?

Reply:
Created one here: https://bitgoinc.atlassian.net/browse/BG-61198